src/inc/AppxPackaging.hpp

```diff
@@ -1044,6 +1061,23 @@ enum tagLOCKTYPE
     };
 #endif 	/* __IAppxBundleManifestPackageInfo_INTERFACE_DEFINED__ */
 
+#ifndef __IAppxManifestOptionalPackageInfo_INTERFACE_DEFINED__
+#define __IAppxManifestOptionalPackageInfo_INTERFACE_DEFINED__
+
+    // {2634847D-5B5D-4FE5-A243-002FF95EDC7E}
+    MSIX_INTERFACE(IAppxManifestOptionalPackageInfo, 0x54cd06c1, 0x268f, 0x40bb, 0x8e, 0xd2, 0x75, 0x7a, 0x9e, 0xba, 0xec, 0x8d); // what are these numbers for?
+    interface IAppxManifestOptionalPackageInfo : public IUnknown

```

Comment:
>IAppxManifestOptionalPackageInfo [](start = 14, length = 32)

For completeness, add IAppxManifestOptionalPackageInfoUtf8.

The sdk also provides utf8 variant interfaces for any interface that has wchar_t  as in or out parameters.

The signature is the same but char is used. 

This is mainly to give support for non-windows platforms.

See other IAppx*Utf8 interface for examples